app/views/articles/index.html.erb

```diff
@@ -1,6 +1,11 @@
 <h1>articles below</h1>
-<%= link_to 'Log in ', new_user_session_path %> </br>
-<%= link_to 'Log out ', destroy_user_session_path(current_user), method: :delete %> </br>
+ <% if user_signed_in? %>
+<%= link_to 'Log out ', destroy_user_session_path, method: :delete %> </br>
```

Comment:
when a method is specified, no need to use `current_user`.
only method: :`delete` will do the job. If `current_user` is used then the devise controller will not render.